recipe/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set name = "scrubadub_address" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LeapBeyond/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: c7679c661cdf2ccca9e39084ea06eb3a47668fc257eda4c2c0a81ee8bdbef3f8
+  patches:
+    - patches/0001-remove-mypy-and-flake8-tests.patch
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # No libpostal for win or s390x
+  # https://github.com/AnacondaRecipes/postal-feedstock/blob/master/recipe/meta.yaml#L15
+  skip: True  # [s390x]
+  skip: True  # [win]
+
+requirements:
+  build: 
+    - patch  # [not win]
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+  run:
+    - python
+    - pyap
+    - scrubadub >=2.0.0rc0
```

No pre-releases, please

```suggestion
    - scrubadub >=2.0.0
```